.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: android
+android:
+  components:
+    - build-tools-25.0.3
+    - android-25
+    - extra-android-m2repository
+  licenses:
```

Comment:
According to the [official documentation](https://docs.travis-ci.com/user/languages/android/#Dealing-with-Licenses), licenses are accepted  automatically, so this line is unnecessary.

Reply:
I tried to get rid of the option but in the in the Travis failed. It seems that the license still needs to be explicitly mentioned in the script. Otherwise, it will fail with an error.

<img width="1110" alt="screen shot 2017-11-24 at 02 22 20" src="https://user-images.githubusercontent.com/758909/33191923-949a8622-d0be-11e7-9e8a-38f0f7a8544e.png">